js/my.js

```diff
@@ -200,10 +200,27 @@ document.addEventListener("DOMContentLoaded", function () {
         }
         msg.addText(from, Utf8.decode(text));
 
-        var img = "https://raw.github.com/nickdesaulniers/fxos-irc/master/128.png";
+        var img = "https://raw.githubusercontent.com/nickdesaulniers/fxos-irc/master/images/128.png";
+
         if ($("container").selectedCard.id !== msg.card.id) {
           // This hack is because origin is not supported in manifests for < 1.1.
-          sendNotification(from, { body: Utf8.decode(text), icon: img });
+          var notif = sendNotification(from, { body: Utf8.decode(text), icon: img });
+
+          notif.onclick = function() {
+              if (document.visibilityState === 'hidden') {
+                  var request = window.navigator.mozApps.getSelf();
+
+                  request.onsuccess = function() {
+                        request.result.launch();
+                        Tab.showTab(host, from);
+                        notif.close();
+                  };
+              } else {
+                  Tab.showTab(host, from);
+                  notif.close();
+              }
+          }
+
```

Comment:
Nice, this looks good.  Just update the indentation to use 2 spaces rather than 4 and this is good to go!